codec-http/src/main/java/io/netty/handler/codec/http/cookie/DefaultCookie.java

```diff
@@ -232,6 +241,9 @@ public String toString() {
         if (isHttpOnly()) {
             buf.append(", HTTPOnly");
         }
+        if (sameSite() != null) {
+            buf.append(", SameSite=").append(sameSite());
```

Comment:
Is this correct? According to [draft](https://tools.ietf.org/html/draft-west-first-party-cookies-07):

> If the "cookie-attribute-list" contains an attribute with an "attribute-name" of "SameSite", set the cookie's "samesite-flag" to "attribute-value" ("Strict" or "Lax").  Otherwise, set the cookie's "samesite-flag" to "None".

So it looks to me "None" is only a possible value in the storage model, not on the wire, and we could use null as well.

Reply:
HI @slandelle  I noticed that discrepancy but that's actually the opposite of what Google Chrome has decided to do. They consider "Lax" the default and for None you need to explictly set it to SameSite=None.
See for instance this article (looks easier to read, but I haven't checked it ): https://blog.chromium.org/2019/10/developers-get-ready-for-new.html
or here: https://www.chromium.org/updates/same-site

That's why it is so urgent for websites to add support for the SameSite attribute, otherwise any third-party cookies without SameSite=None will be automatically rejected by Chrome.

Reply:
Ah, actually this was defined here: https://tools.ietf.org/html/draft-west-cookie-incrementalism-00#section-3.1